File: app/src/main/java/id/ac/ui/cs/mobileprogramming/jeremy/lab07/MainActivity.java
package id.ac.ui.cs.mobileprogramming.jeremy.lab07;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private int count = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button increaseButton = findViewById(R.id.button);
        TextView counter = findViewById(R.id.counter);

        increaseButton.setOnClickListener(v ->{
            count+=1;
            counter.setText(Integer.toString(count));
        });
    }
}

File: app/src/main/java/id/ac/ui/cs/mobileprogramming/jeremy/lab02/MainActivity.kt
package id.ac.ui.cs.mobileprogramming.jeremy.lab02

import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import android.os.Bundle
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    var counter = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        value.setText("" + counter)

        plusBtn.setOnClickListener {
            value.setText("" + ++counter)
        }

        minusBtn.setOnClickListener {
            value.setText("" + --counter)
        }

        if (counter == 10){
            onTen()
        }
    }

    private fun onTen(){
        val counterText: TextView = findViewById(R.id.counterText)
        counterText.text = counterTextMessage()
    }
    fun counterTextMessage(): String{
        return "You made it to 10!"
    }

}